crates/types/src/traits/block_contents.rs

```diff
@@ -102,6 +97,12 @@ pub fn vid_commitment(
     vid.commit_only(encoded_transactions).unwrap()
 }
 
+/// Computes the (empty) genesis VID commitment
+#[must_use]
+pub fn genesis_vid_commitment() -> <VidScheme as VidSchemeTrait>::Commit {
+    vid_commitment(&vec![], 8)
```

Comment:
Pretty sure we don't want to compute the genesis commitment like this but there's no need to worry about that now.